### PR TITLE
Deprecated issue fixed #453 

### LINKED ISSUE #453: DEPRECATED - FormTypeInterface::setDefaultOptions()
DEPRECATED - Vich\UploaderBundle\Form\Type\VichFileType: The FormTypeInterface::setDefaultOptions() method is deprecated since version 2.7 and will be removed in 3.0. Use configureOptions() instead. This method will be added to the FormTypeInterface with Symfony 3.0.

### PR DESCRIPTION
**Pull request by bobod**
composer.json updated because of PHPUnit test. It depends on the added packages.
#453

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     },
     "require": {
         "php": ">=5.4",
-        "symfony/framework-bundle": "~2.3",
+        "symfony/framework-bundle": "~2.7.5",
```
Comment (K-Phoen) [inline composer.json]: Hum, it means that all Symfony versions prior to 2.7.5 would not be supported by this bundle. That's a problem, we do want to support them.

Comment (bobod) [inline composer.json]: Yes, it's just a mistake, because I had to add some packages to the composer.json in local environment in order to run unit tests.

**Comment by K-Phoen** [comment]
This PR looks good to me. There is no BC break and I trust you when you say that it solves the deprecation warning.

Could you [squash your commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)? I'll merge the PR once it's done.

**Comment by bobod** [comment]
This was the first time when I squashed my commits, but I think I've done with it.

**Comment by K-Phoen** [comment]
Merged by [this commit](https://github.com/dustin10/VichUploaderBundle/commit/27888d5aa2e6676fbfb565dd47eac8c538735400).

Thanks :+1:

**Pull request closed without merging**